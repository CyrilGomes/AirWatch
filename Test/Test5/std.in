1
government@mail.com
123456
thisIsAString
10
1
5
4
3
