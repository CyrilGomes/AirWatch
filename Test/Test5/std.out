
=========================
 Welcome to AirWatch :
=========================

1. Login
2. Register
3. Exit
> :
Login
-----

Email : Password :
=========================
 Main menu :
=========================

1. Sensor analytics
2. Cleaner analytics
3. Register Air Cleaning Company
4. Log out
> : > : > :
=========================
 Sensor analytics menu :
=========================

1. Get the air quality of a given location
2. Get the air quality of a given area
3. See sensor reliabilities
4. Find data similarities
5. Go back
> :
=========================
 Main menu :
=========================

1. Sensor analytics
2. Cleaner analytics
3. Register Air Cleaning Company
4. Log out
> :
=========================
 Welcome to AirWatch :
=========================

1. Login
2. Register
3. Exit
> :
